### ISSUE TITLE
Flaky Jest test: `<VerificationQRCode /> › renders a QR code`

### ISSUE THREAD

**Issue by richvdh** [element-hq/element-web#33219]
https://github.com/element-hq/element-web/actions/runs/24713591287/job/72284161042?pr=33216#step:8:75453
```
    ● <VerificationQRCode /> › renders a QR code
      Unable to find an element with the alt text: QR Code
      Ignored nodes: comments, script, style
      <body>
        <div>
          <div
            class="mx_QRCode mx_VerificationQRCode"
          >
            <div
              class="mx_Spinner"
            >
              <svg
                aria-label="Loading…"
                class="_icon_1855a_18"
                data-testid="spinner"
                fill="currentColor"
                height="1em"
                role="progressbar"
                style="width: 32px; height: 32px;"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M12 4.031a8 8 0 1 0 8 8 1 1 0 0 1 2 0c0 5.523-4.477 10-10 10s-10-4.477-10-10 4.477-10 10-10a1 1 0 1 1 0 2"
                  fill-rule="evenodd"
                />
              </svg>
            </div>
          </div>
        </div>
      </body>
  
        22 |         );
        23 |         // wait for the spinner to go away
      > 24 |         await waitFor(() => getAllByAltText("QR Code").length === 1);
           |                      ^
        25 |         expect(container).toMatchSnapshot();
        26 |     });
        27 | });
  
        at waitForWrapper (../../node_modules/@testing-library/dom/dist/wait-for.js:163:27)
        at Object.<anonymous> (test/unit-tests/components/views/elements/crypto/VerificationQRCode-test.tsx:24:22)
```
